docs/administration/manage-dependencies/message-storage.md

```diff
@@ -238,8 +238,8 @@ metadata:
 spec:
   config:
     kafka:
-      # securityPolicy supports: PLAINTEXT, SSL, SASL_PLAINTEXT, SASL_SSL 
-      securityPolicy: PLAINTEXT
+      # SecurityProtocol supports: PLAINTEXT, SSL, SASL_PLAINTEXT, SASL_SSL 
+      SecurityProtocol: PLAINTEXT
```

Comment:
Another catch XD, but the fields in `spec.config` are capital sensitive, see:
https://github.com/milvus-io/milvus/blob/v2.3.7/pkg/util/paramtable/service_param.go#L688